gradle.properties

```diff
@@ -8,20 +8,20 @@ readable_name=FTB Chunks
 maven_group=dev.ftb.mods
 mod_author=FTB Team
 
-mod_version=2100.1.0
+mod_version=2100.1.1
 minecraft_version=1.21
 
 # Deps
 #forge_version=50.0.9
-neoforge_version=21.0.10-beta
+neoforge_version=21.0.109-beta
```

Comment:
@desht do you want to push your range changes in here?

Reply:
yep good idea